1-exercises/E-for-loop/exercise2.js

```diff
@@ -26,6 +26,13 @@ const AGES = [
     49
 ];
 
+let i = 0;
+
```

Comment:
See comments above. In a while loop, we initialise the index variable above the while loop statements. In a for loop, the index variable is initialised within the for loop statement brackets.